### PR TITLE
New json report feature

### PR DESCRIPTION
**Pull request by mariangemarcano**

### CODE REVIEW

File: www/Gruntfile.js [hunk 1]
```diff
@@ -47,28 +52,30 @@ module.exports = function (grunt) {
...
-                    environment: "development",
-                    outputStyle: "nested"
-                }
+                config: "<%= files.fonts.config %>",
+                overwrite: true,
+                fonts: "<%= files.fonts.dist %>"
             }
         },
+        sass: {
+			options: {
```
Comment (opdude) [inline www/Gruntfile.js]: Indented incorrectly?

File: www/sass/modules/framework/_typography.scss [hunk 2]
```diff
@@ -6,11 +6,24 @@
...
 */
 
-@include font-face("katana_icons", font-files("katana_icons/katana_icons.eot?cachebusting=6", "katana_icons/katana_icons.ttf?cachebusting=6"));
-@include font-face("pacifico", font-files("pacifico/pacifico-webfont.eot?cachebusting=2", "pacifico/pacifico-webfont.ttf?cachebusting=2"));
-@include font-face("leckerlione", font-files("leckerlione/leckerlione-regular-webfont.eot?cachebusting=6", "leckerlione/leckerlione-regular-webfont.ttf?cachebusting=6"));
-    
-  
+@font-face {
+  font-family: "katana_icons";
+  src: url("../../fonts/katana_icons/katana_icons.eot?cachebusting=6") format('eot'),
```
Comment (opdude) [inline www/sass/modules/framework/_typography.scss]: Should we update the cachebusting as we changed it?

File: www/sass/modules/page-elements/_search-drop-down.scss [hunk 3]
```diff
@@ -54,7 +54,7 @@ div.common-branch-select {
 
 .select2-container {
 	min-width:90px;
```
Comment (opdude) [inline www/sass/modules/page-elements/_search-drop-down.scss:56]: Incorrect indentation

File: www/Gruntfile.js (hunk 1 above)
Comment (kkateq) [inline www/Gruntfile.js]: yes, incorrect, but I am not considering this is a major issue and wouldn't fix it

File: www/sass/modules/framework/_typography.scss (hunk 2 above)
Comment (kkateq) [inline www/sass/modules/framework/_typography.scss]: this should be fixed

File: www/sass/modules/page-elements/_search-drop-down.scss (hunk 3 above)
Comment (kkateq) [inline www/sass/modules/page-elements/_search-drop-down.scss:56]: This file full of incorrect indentation, mixed two and four spaces, I would leave this fix for future

**Pull request merged**